packages/react-client/src/components/coffee/AddCoffeeForm.tsx

```diff
@@ -105,21 +111,13 @@ const AddCoffeeForm = ({ userId }: AddCoffeeFormProps) => {
         />
       </StyledLabel>
       <StyledLabel>
-        Rating
-        {/*
-            @todo Replace Select options with CoffeeStars
-            @body Updat the below select to use the coffee star component built in #25
-          */}
-        <select
-          value={coffee.rating}
-          onChange={event => setCoffee({ ...coffee, rating: parseInt(event.target.value, 10) })}
-        >
-          {coffeeScores.map(score => (
-            <option key={score} value={score}>
-              {score}
-            </option>
-          ))}
-        </select>
+        <StyledComponent>
+          Rating <br />
+          <CoffeeStars
+            value={coffee.rating}
+            onChange={(event: number) => setCoffee({ ...coffee, rating: event })}
```

Comment:
instead of `event: number` you could do `rating: number` and then you can just do `setCoffee({...coffee, rating})` but that's all good

Reply:
Always forget about this. Will keep it in mind!